packages/wdio-cli/src/utils.ts

```diff
@@ -287,6 +294,17 @@ export function getCapabilities(arg: ReplCommandArguments) {
         return { capabilities: { browserName: 'Chrome', ...ANDROID_CONFIG, ...optionalCapabilites } }
     } else if (/ios/.test(arg.option)) {
         return { capabilities: { browserName: 'Safari', ...IOS_CONFIG, ...optionalCapabilites } }
+    } else if (/(js|ts)$/.test(arg.option)) {
+        const config = new ConfigParser()
+        config.autoCompile()
+        config.addConfigFile(arg.option)
+        let requiredCaps = config.getCapabilities()
+        if (Object.prototype.hasOwnProperty.call(arg, 'capabilities')) {
+            requiredCaps = (requiredCaps as (DesiredCapabilities | W3CCapabilities)[])[Number(arg.capabilities)] ||
+                (requiredCaps as MultiRemoteCapabilities)[arg.capabilities]
+        }
+        const requiredW3CCaps = pickBy(requiredCaps, (_, key) => !IGNORED_CAPABILITIES.includes(key))
+        return { capabilities: { ...(requiredW3CCaps as W3CCapabilities) } }
```

Comment:
before we return, let's make sure this object exists and if not (e.g. a user picks a capability that doesn't exist) throw an error

Reply:
@christian-bromann  the webdriver session throws the error saying capability is invalid, should we explicitly throw ?

Reply:
No, what I meant was, let's say the capabilities are these:

```
// wdio.conf.js
exports.config = {
  // ...
  capabilities: [{ ... }] 
  // ...
}
```

and someone runs:

```sh
npx wdio repl "./wdio.conf.js" 123
```

Given that there is no capability that can be index with 123, we should throw an error.